### DISCUSSION TITLE
Is it possible to implement smooth scrolling?

### DISCUSSION THREAD

**Discussion by Andre601** [squidfunk/mkdocs-material#2183] [Q&A] [answered]
One thing I currently don't like about the theme (And maybe MkDocs as a whole?) is, that when you click on a link, which points to a header, or when you click a header's anchor, that it "snaps" to this section.

Is there a way to change this with smooth scrolling, so that instead of you instantly landing at that section, you would instead smoothly scroll up/down to it?

**Comment by squidfunk** [comment, marked as answer]
This question was already answered in #2128.

**Reply by Andre601** [reply]
Was looking at it, but seems it doesn't work for me?
My browser for sure does support it, but `scrolling-behaviour: smooth;` inside the html section sure does nothing here.

Is this here really everything or would I need to look for more?
https://github.com/purrbot-site/Docs/blob/1ae56fc9dfa6d27310796fa0d7c0beed774a955c/docs/assets/css/styling/colors.css#L10-L12

**Reply by squidfunk** [reply]
Just saw this on Twitter - here's is why you probably don't want smooth scrolling:
https://twitter.com/chriscoyier/status/1346513455516426242

**Reply by Andre601** [reply]
That makes sense.
